infra/src/main/java/com/redhat/service/smartevents/infra/exceptions/mappers/ConstraintViolationExceptionMapper.java

```diff
@@ -76,7 +73,7 @@ public ErrorResponse apply(ConstraintViolation<?> cv) {
         private ErrorResponse unmappedConstraintViolation(ConstraintViolation<?> cv) {
             Optional<BridgeError> error = bridgeErrorService.getError(UnclassifiedConstraintViolationException.class);
```

Comment:
@manstis I'd suggest that we simply fail to boot if a `BridgeError` that is absolutely required is missing. That should stop us getting anywhere near our `stage`/`prod` environments with missing entries in `exceptionInfo.json`.